src/main/java/beam/agentsim/events/handling/BeamEventsLogger.java

```diff
@@ -51,8 +51,26 @@ public BeamEventsLogger(BeamServices beamServices, EventsManager eventsManager)
         allLoggableEvents.add(PersonLeavesVehicleEvent.class);
         allLoggableEvents.add(PersonArrivalEvent.class);
         allLoggableEvents.add(ActivityStartEvent.class);
-
+        //Adding attribute which should be removed when the logger level SHORT
         eventFieldsToDropWhenShort.put(PathTraversalEvent.class,PathTraversalEvent.ATTRIBUTE_VIZ_DATA);
+        eventFieldsToDropWhenShort.put(PathTraversalEvent.class,PathTraversalEvent.ATTRIBUTE_LINK_IDS);
```

Comment:
eventFieldsToDropWhenShort and eventFieldsToAddWhenVerbose should be multimaps. If you are unfamiliar with this data type:

http://google.github.io/guava/releases/23.0/api/docs/com/google/common/collect/Multimap.html